spring-cloud-skipper-server-core/src/test/resources/repositories/sources/test/ticktock/ticktock-1.0.1/package.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: skipper.spring.io/v1
```

Comment:
Given these package files are updated in the `sources`, the corresponding package zip files need to be updated as well.

Reply:
The ticktock-1.0.1 resource is used only by the ManifestUtilsTests. Not sure if we need zip packages for it?

Reply:
I meant this: https://github.com/spring-cloud/spring-cloud-skipper/tree/master/spring-cloud-skipper-server-core/src/test/resources/repositories/binaries/test/ticktock

We still use to test things by uploading this package.

Reply:
Yes, the package .zips should match the contents of the zipped directories.

Reply:
However, we never load a ticktock-1.0.1.zip....  So maybe the tests should change to use a new ticktock-1.0.1.zip instead of ticktock-1.0.0.zip